### DISCUSSION TITLE
Zweites Menu

### DISCUSSION THREAD

**Discussion by ma-gu-16** [Zefau/ioBroker.jarvis#2645] [Q&A] [answered]
Ist es möglich bei den Tabs eine Art zusätzliches Menu zu machen, welches auf die Seiten links verlinkt? 

Das komplette Tab zu kopieren halte ich für extrem redundant. 

![image](https://github.com/Zefau/ioBroker.jarvis/assets/22169074/85f15c9c-47a3-4aea-8dc6-d74b2e758ae0)

Wie habt ihr das gelöst? 

Danke euch.

**Comment by mcuiobroker** [comment]
Könntest du es ein wenig weiter ausführen, was du erreichen willst?
Auf der Übersicht-Seite sollen neben dem Überblick-TAB , weitere Tabs, die dann auf andere TABs der anderen Seiten verlinken?
Du kannst sie doch einfach über die Seitenauswahl wählen.
Was soll das bringen?
Mit HomeKitTile kannst du auf der Übersicht auch Links festlegen.

**Reply by ma-gu-16** [reply]
ES würde ein Klick sparen. Je nach Rahmen in der Wand ist es etwas umständlich den Hamburger in der Seite zu klicken. (hab den Link via CSS schon grösser gemacht)

So hätte ich 2-3 "wichtige" Seiten direkt auf dem Homescreen verlinkt.  

Ich schau mir mal das HomeKitTile an, evtl lässt es sich damit ja lösen. 

Danke dir.

**Comment by ma-gu-16** [comment]
Also die Funktion des HomeKitTile wäre fast das Richtige. Nimmt nur sehr viel Platz weg für ein einfachen Link. Zumal wenn ich eben SEiten und Tabs ebenso aktiv habe. 

Wenn ich jetzt in 2. Reihe anfange die Tiles zu machen, sieht es optisch eher nciht so gut aus und nimmt zuviel Platz. 

Evlt lege ich später ein Feature Request an.

**Reply by mcuiobroker** [reply]
> Nimmt nur sehr viel Platz weg für ein einfachen Link.

Kannst du doch in der Größe einstellen.
Alterantiv gibt es Zusatzprogramme
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/scripts-3.1.x-pro/homekittile/hkt-bis-zu-9-dps-boolean-umschalten
![image](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/0e79480c-0e51-4517-a804-c2b7a818d557)

**Reply by ma-gu-16** [reply]
> > Nimmt nur sehr viel Platz weg für ein einfachen Link.
> 
> Kannst du doch in der Größe einstellen. Alterantiv gibt es Zusatzprogramme https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-

das ist immernoch viel ;) vorallem wenn man bedenkt das ja oben bei den Tabs der platz zur Verfügung stehen würde. 

![image](https://github.com/Zefau/ioBroker.jarvis/assets/22169074/5314bd06-fc83-4daf-94cc-585419263587)

Neben "Überblick" ein "Zu den Kameras" wäre toll, zumal es sich ja irgendwo auch um eine Art Navigation handelt.

**Comment by ma-gu-16** [comment]
Schön wäre evtl auch wenn men das Seiten Menu immer offen haben lassen kann, je nach Device, also entweder nur die Icons oder Icons mit Text. 

![image](https://github.com/Zefau/ioBroker.jarvis/assets/22169074/694737d9-370d-4866-969d-37990ced47fd)

ungefähr so. Jedoch klappt das komplett weg auf dem Tablet.

**Reply by mcuiobroker** [reply]
> Jedoch klappt das komplett weg auf dem Tablet.

Bedeutet? Bild.

**Reply by ma-gu-16** [reply]
> > Jedoch klappt das komplett weg auf dem Tablet.
> 
> Bedeutet? Bild.

![image](https://github.com/Zefau/ioBroker.jarvis/assets/22169074/f855e96b-3c2b-4a57-9590-c150090c9122)

**Reply by mcuiobroker** [reply]
Das ist dann ein Bug und sollte auch entsprechend in den Issues benannt werden.

**Reply by mcuiobroker** [reply]
Dies Problem trotzdem bitte als BUG in die Issues eintragen. Danke.

**Comment by mcuiobroker** [comment, marked as answer]
Mit tabLink (Zusatzprogramm scripts)
![tabLink](https://github.com/Zefau/ioBroker.jarvis/assets/84818934/fa76ae8c-37e9-4cbb-85e4-a1551246409f)

**Reply by ma-gu-16** [reply]
ah yes. genau sowas :) wo finde ich das zusatz script ?

**Reply by mcuiobroker** [reply]
Muss ich noch erst in der Doku reinstellen, dauert noch etwas.

**Reply by ma-gu-16** [reply]
Ok danke dir, wäre froh wenn du hier einfach kommentierst damit ich das mitbekomme ;) Grüsse

**Reply by mcuiobroker** [reply]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis-v3/besonderheiten-v3/scripts-3.1.x-pro/tab/tablink
